[BUG] not convenient to debug e2e-test locally

During local development stage, as I just want to debug one test case, like `test/e2e-test/component_version_test.go`, I will force close the whole test process `suit_test.go`. Then I have to do the following steps to start a fresh round, which take a lot of time for a single test.

##  `suite_test.go` again.
```
GOROOT=/usr/local/go #gosetup
GOPATH=/Users/zhouzhengxi/Programming/golang #gosetup
/usr/local/go/bin/go test -c -o /private/var/folders/g5/mzgjztlj1sq7mwvltkl81_rr0000gp/T/___suite_test_go__1_ -gcflags all=-N -l github.com/crossplane/oam-kubernetes-runtime/test/e2e-test #gosetup
/usr/local/go/bin/go tool test2json -t /Applications/GoLand.app/Contents/plugins/go/lib/dlv/mac/dlv --listen=0.0.0.0:61981 --headless=true --api-version=2 --check-go-version=false --only-same-user=false exec /private/var/folders/g5/mzgjztlj1sq7mwvltkl81_rr0000gp/T/___suite_test_go__1_ -- -test.v -test.run ^TestAPIs$ #gosetup
API server listening at: [::]:61981
=== RUN   TestAPIs
Running Suite: OAM Core Resource Controller Suite
=================================================
Random Seed: 1605601441
Will run 13 of 13 specs

I1117 16:24:02.765973   74567 request.go:621] Throttling request took 1.019589954s, request: GET:https://127.0.0.1:51736/apis/autoscaling/v1?timeout=32s
STEP: Bootstrapping test environment
STEP: Setting up kubernetes client
STEP: Finished setting up test environment
STEP: Created manual scalar trait definition
STEP: Created extended manualscalertraits.core.oam.dev
STEP: Created containerizedworkload.core.oam.dev
STEP: Created example.com cluster role for the test service account
Failure [1.231 seconds]
[BeforeSuite] BeforeSuite 
/Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:69

  Expected
      <*errors.StatusError | 0xc0003743c0>: {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "clusterrolebindings.rbac.authorization.k8s.io \"oam-role-binding\" already exists",
              Reason: "AlreadyExists",
              Details: {
                  Name: "oam-role-binding",
                  Group: "rbac.authorization.k8s.io",
                  Kind: "clusterrolebindings",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 409,
          },
      }
  to be nil

  /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:198
------------------------------


Ran 13 of 0 Specs in 1.307 seconds
FAIL! -- 0 Passed | 13 Failed | 0 Pending | 0 Skipped
--- FAIL: TestAPIs (1.31s)
FAIL

Debugger finished with exit code 0

```

## run one more time to clear resources
```
GOROOT=/usr/local/go #gosetup
GOPATH=/Users/zhouzhengxi/Programming/golang #gosetup
/usr/local/go/bin/go test -c -o /private/var/folders/g5/mzgjztlj1sq7mwvltkl81_rr0000gp/T/___suite_test_go__1_ -gcflags all=-N -l github.com/crossplane/oam-kubernetes-runtime/test/e2e-test #gosetup
/usr/local/go/bin/go tool test2json -t /Applications/GoLand.app/Contents/plugins/go/lib/dlv/mac/dlv --listen=0.0.0.0:62022 --headless=true --api-version=2 --check-go-version=false --only-same-user=false exec /private/var/folders/g5/mzgjztlj1sq7mwvltkl81_rr0000gp/T/___suite_test_go__1_ -- -test.v -test.run ^TestAPIs$ #gosetup
API server listening at: [::]:62022
=== RUN   TestAPIs
Running Suite: OAM Core Resource Controller Suite
=================================================
Random Seed: 1605601536
Will run 13 of 13 specs

I1117 16:25:38.007933   74682 request.go:621] Throttling request took 1.031501846s, request: GET:https://127.0.0.1:51736/apis/cert-manager.io/v1beta1?timeout=32s
STEP: Bootstrapping test environment
STEP: Setting up kubernetes client
STEP: Finished setting up test environment
Failure [3.719 seconds]
[BeforeSuite] BeforeSuite 
/Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:69

  Expected
      <*meta.NoKindMatchError | 0xc00053f800>: {
          GroupKind: {
              Group: "core.oam.dev",
              Kind: "TraitDefinition",
          },
          SearchedVersions: ["v1alpha2"],
      }
  To satisfy at least one of these matchers: [%!s(*matchers.BeNilMatcher=&{}) %!s(*util.AlreadyExistMatcher=&{})]

  /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:111
------------------------------
STEP: Tearing down the test environment
Failure [0.009 seconds]
[AfterSuite] AfterSuite 
/Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:248

  Expected
      <*errors.StatusError | 0xc00032c1e0>: {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "clusterrolebindings.rbac.authorization.k8s.io \"oam-role-binding\" not found",
              Reason: "NotFound",
              Details: {
                  Name: "oam-role-binding",
                  Group: "rbac.authorization.k8s.io",
                  Kind: "clusterrolebindings",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 404,
          },
      }
  to be nil

  /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/test/e2e-test/suite_test.go:256
------------------------------


Ran 13 of 0 Specs in 3.729 seconds
FAIL! -- 0 Passed | 13 Failed | 0 Pending | 0 Skipped
--- FAIL: TestAPIs (3.73s)
FAIL

Debugger finished with exit code 0

```

## install crds
```
➜  /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/charts/oam-kubernetes-runtime/crds git:(issue-revision) ✗ k apply -f .
customresourcedefinition.apiextensions.k8s.io/applicationconfigurations.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/components.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/containerizedworkloads.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/healthscopes.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/manualscalertraits.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/scopedefinitions.core.oam.dev configured
customresourcedefinition.apiextensions.k8s.io/traitdefinitions.core.oam.dev created
customresourcedefinition.apiextensions.k8s.io/workloaddefinitions.core.oam.dev created
```

## tigger the normal e2e-test
run `test_suit.go` and wait the reconciler.

--- a/test/e2e-test/suite_test.go
+++ b/test/e2e-test/suite_test.go
@@ -94,6 +94,15 @@ var _ = BeforeSuite(func(done Done) {
 	}
 	By("Finished setting up test environment")
 
+	By("Applying CRD of WorkloadDefinition and TraitDefinition")
+	var workloadDefinitionCRD crdv1.CustomResourceDefinition
+	Expect(readYaml("../../charts/oam-kubernetes-runtime/crds/core.oam.dev_workloaddefinitions.yaml", &workloadDefinitionCRD)).Should(BeNil())
+	Expect(k8sClient.Create(context.Background(), &workloadDefinitionCRD)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
+
+	var traitDefinitionCRD crdv1.CustomResourceDefinition
+	Expect(readYaml("../../charts/oam-kubernetes-runtime/crds/core.oam.dev_traitdefinitions.yaml", &traitDefinitionCRD)).Should(BeNil())
+	Expect(k8sClient.Create(context.Background(), &traitDefinitionCRD)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
+
 	// Create manual scaler trait definition
 	manualscalertrait = v1alpha2.TraitDefinition{
 		ObjectMeta: metav1.ObjectMeta{

